k8s/default/kustomization.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-app-service
+  namespace: default
+spec:
+  selector:
+    app: test-app
+  ports:
+  - port: 80
+    targetPort: 8080
+  type: ClusterIP
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)